dev/packages/beats/nginx-0.0.1/docs/README.md

```diff
@@ -1,15 +1,127 @@
 # Nginx Integration
 
-TODO
+This integration periodically fetches metrics from [https://nginx.org/](Nginx) servers.
+
 
 ## Compatibility
 
-TODO
+The Nginx stubstatus metrics were tested with Nginx 1.9 and are expected to work with all version >= 1.9. The logs were tested with version 1.10. On Windows, the module was tested with Nginx installed from the Chocolatey repository.
+
+
+## Logs
+
+
+**Timezone support**
+
+This datasource parses logs that don’t contain timezone information. For these logs, the Elastic Agent reads the local timezone and uses it when parsing to convert the timestamp to UTC. The timezone to be used for parsing is included in the event in the event.timezone field.
+
+To disable this conversion, the event.timezone field can be removed with the drop_fields processor.
+
+If logs are originated from systems or applications with a different timezone to the local one, the event.timezone field can be overwritten with the original timezone using the add_fields processor.
+
+### Access Logs
+
+Access logs collects the nginx access logs.
+
+**Exported fields**
+
+| Field | Description | Type |
+|---|---|---|
+| nginx.access.agent | Alias for field "user_agent.original" | alias |
+| nginx.access.body_sent.bytes | Alias for field "http.response.body.bytes" | alias |
```

Comment:
These alias fields are not part of the even anymore. These were only used for migration from 6.x to 7.x and are not needed anymore.

Reply:
Based on the offline discussion - I will skip all the alias fields with the `migration: true` property.

Reply:
Fixed.